sql/moz-fx-data-shared-prod/telemetry_derived/sponsored_tiles_clients_daily_v1/query.sql

```diff
@@ -331,7 +334,8 @@ SELECT
   COALESCE(sponsored_tiles_click_count, 0) AS sponsored_tiles_click_count,
   COALESCE(sponsored_tiles_impression_count, 0) AS sponsored_tiles_impression_count,
   NULL AS sponsored_tiles_dismissal_count,
-  COALESCE(sponsored_tiles_disable_count, 0) AS sponsored_tiles_disable_count
+  COALESCE(sponsored_tiles_disable_count, 0) AS sponsored_tiles_disable_count,
+  NULL AS profile_group_id
```

Comment:
it might be worth casting this `NULL` to a string explicitly. By default `NULL` is considered an integer in BigQuery.

Reply:
OMG thank you for reminding me, I literally did this mistake just the other day too - good catch